src/js/cf-tables.js

```diff
@@ -0,0 +1,162 @@
+
+/**
+ * cf-tables
+ * https://github.com/cfpb/cf-tables
+ *
+ * A public domain work of the Consumer Financial Protection Bureau
+ */
+
+( function( $ ) {
+  'use strict';
+
+  var SortableTable = function( table, options ) {
+    /*  At the moment, there are no default settings, but here's an object
+        for their future use! */
+    var defaults = {
+        },
+    // settings is defaults combined with user options
+    settings = {},
+    // rows is an Array of Arrays, serving as a model of the table
+    rows = [],
+    $table = $( table ),
+    $headercells = $( table ).find( '.sortable' );
```

Comment:
Good catch. I'll just change the line to `$headercells = $table.find( '.sortable' );` - the `this.` shouldn't be necessary.